source/cloud/azure/azureml.md

```diff
@@ -55,9 +65,165 @@ Launch the instance.
+
+1. Select **Data** from the left-hand menu > **Datastores** > choose your datastore name > **Browse**
+2. Find the file/folder containing your dataset and click the elipsis (...) next to it.
+3. From the menu, choose **Copy URI** and select **Datastore URI** format to copy into your notebook.
+
+![Screenshot of access datastore uri screen](../../images/azureml-access-datastore-uri.png)
+
+### Custom RAPIDS Environment
+
+To run an AzureML experiment, you must specify an [environment](https://learn.microsoft.com/en-us/azure/machine-learning/concept-environments?view=azureml-api-2) that contains all the necessary software dependencies to run the training script on distributed nodes. <br>
+You can define an environment from a [pre-built](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-docker-image) docker image or create-your-own from a [Dockerfile](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-docker-build-context) or [conda](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-environments-v2?tabs=python&view=azureml-api-2#create-an-environment-from-a-conda-specification) specification file.
+
+Create your custom RAPIDS docker image using the example below, making sure to install additional packages needed for your workflows.
+
+```dockerfile
+
+# Use latest rapids image with the necessary dependencies
+FROM {{ rapids_container }}
+
+# Update and/or  install required packages
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential fuse && \
+    rm -rf /var/lib/apt/lists/*
+
+# Activate rapids conda environment
+RUN /bin/bash -c "source activate rapids && pip install azureml-mlflow"
+```
+
+Now create the Environment, making sure to label and provide a description:
+
+```python
+from azure.ai.ml.entities import Environment, BuildContext
+
+env_docker_image = Environment(
+    build=BuildContext(path="Dockerfile"),
+    name="rapids-mlflow",
+    description="RAPIDS environment with azureml-mlflow",
+)
+
+ml_client.environments.create_or_update(env_docker_image)
+```
+
+### Submit RAPIDS Training jobs
+
+Now that we have our environment and custom logic, we can configure and run the `command` [class](https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml?view=azure-python#azure-ai-ml-command) to submit training jobs. `inputs` is a dictionary of command-line arguments to pass to the training script.
+
+```python
+from azure.ai.ml import command, Input
+from azure.ai.ml.sweep import Choice, Uniform
+
+command_job = command(
+    environment="rapids-mlflow:1",  # specify version of environment to use
+    experiment_name="test_rapids_mlflow",
+    code=project_folder,
+    command="python train_rapids.py --data_dir ${{inputs.data_dir}} \
```

Comment:
The `inputs` variable is causing build-docs failures.. i think it assumed that the values should be populated in the `version_template` function as is the case for `{{ rapids_container }}`

Not sure how to resolve this, thoughts?

Reply:
I've opened #215 to fix this. Once it's merged you'll need to pull in from `main`.